apps/extension/src/tvm_ext.cc

```diff
@@ -19,12 +22,57 @@ template<>
 struct extension_class_info<tvm_ext::IntVector> {
   static const int code = 17;
 };
+template<>
+struct array_type_index<tvm_ext::NDSubClass> {
+  static const int code = 1;
+};
 }  // namespace tvm
 }  // namespace runtime
 
 using namespace tvm;
 using namespace tvm::runtime;
 
+namespace tvm_ext {
+class NDSubClass : public tvm::runtime::NDArray {
```

Comment:
add some comments on what need to do be done by adding a subclass, and what are the benefit

Reply:
I will copy the description of this PR here.